src/docs/modules/ROOT/index.adoc

```diff
@@ -0,0 +1,35 @@
+= Welcome to the Open Manufacturing Platform - Semantic Data Structuring Documentation
+:page-layout: tiles
```

Comment:
I don't think the sds-antora-ui already includes a page style for tiles...

Reply:
I guess you are right. 
What do you think about keeping this reference to a style for tiles in this PR?

In a next step, we could add the style to the SDS Antora UI, do a release of the SDS Antora UI, and then reference the newer version of the SDS Antora UI which then includes a tile style. In the meantime, the website for the SDS documentation is of course not able evaluate the style but style functions properly.